content/images/github.svg

```diff
@@ -1,6 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- Source: https://github.com/simple-icons/simple-icons/blob/95e3588af989ecf62f5f970af8fc7c64103ecd99/icons/github.svg -->

```

Comment:
Do we want to leave an embedded comment with the new source?

Reply:
Perhaps. In the `manubot-resources` repo I give attribution to Font Awesome, and say that all icons are by them unless otherwise noted. They don't include a url in their svgs by default; they keep it clean and to a bare minimum. Maybe we could include the url in the svgs that are not from FA.

Reply:
My preference is to keep the attribution in the file.

Reply:
Added the link back in. I also changed the fill color to `currentColor`. This will allow us to specify the color in the CSS instead of in the svg, and thus we can tweak the color for future themes without duplicating the svg files.

Reply:
Let's add that change to this PR

Reply:
> `{.author_icon}`

Will wait for the `default-theme` PR to change this this, since without the `{height="13px" width="13px"}`, and since the `.author_icon` CSS class isn't defined yet, the svg will be huge: it'll scale to the full width of the page.